content/docs/command-reference/add.md

```diff
@@ -145,28 +145,31 @@ not.
   [pattern](https://docs.python.org/3/library/glob.html) specified in `targets`.
   Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`, and `**`
 
-- `--external` - allow `targets` that are outside of the DVC repository. See
+- `-o <path>`, `--out <path>` - destination `path` inside the workspace to place
+  a data target (instead of using the current working directory). Directories
+  specified in the path will be created by this command. Note that this can be
```

Comment:
> Note that this can be  used [with an external cache](#straight-to-cache) to avoid using the local   file system.

it's not very clear ...

This can be especially useful when you use an external cache setup with symlinks. To add large data from an external (to the repo) location w/o taking or requiring enough space ...

Reply:
do we need to start with `allow a target outside of the DVC repository` as we do in `--to-remote`?

Reply:
I updated this to "Note that for external targets, this can be combined with an external cache to skip the local file system." when addressing previous feedback. Seems to make enough sense like that?

The link config info seems too detailed and its explained in the link, which people will need to go to anyway for this I think.

> do we need to start with allow a target outside of the DVC repository

Reply:
> do we need to start with allow a target outside of the DVC repository

--out works for local targets too (renames them).

Reply:
That said I'm not sure here. Feels like `add -o` is almost totally different with local vs external targets so yeah, I could write a longer explanation here as a 2nd paragraph including info about links and allowing external targets.